Uruguay/URY_transform_load_dlt.py.py

```diff
@@ -0,0 +1,112 @@
+# Databricks notebook source
+# Databricks notebook source
+import dlt
+import unicodedata
+from pyspark.sql.functions import substring, col, lit, when, udf, trim, regexp_replace, initcap, concat
+from pyspark.sql.types import StringType
+
+# Note DLT requires the path to not start with /dbfs
+TOP_DIR = "/mnt/DAP/data/BOOSTProcessed"
+INPUT_DIR = f"{TOP_DIR}/Documents/input/Countries"
+WORKSPACE_DIR = f"{TOP_DIR}/Workspace"
+COUNTRY = 'Uruguay'
+COUNTRY_MICRODATA_DIR = f'{WORKSPACE_DIR}/microdata_csv/{COUNTRY}'
+
+CSV_READ_OPTIONS = {
+    "header": "true",
+    "multiline": "true",
+    "quote": '"',
+    "escape": '"',
+}
+
+@dlt.expect_or_drop("year_not_null", "YEAR IS NOT NULL")
+@dlt.table(name=f'ury_boost_bronze')
+def boost_bronze():
+    # Load the data from CSV
+    return (spark.read
+            .format("csv")
+            .options(**CSV_READ_OPTIONS)
+            .option("inferSchema", "true")
+            .load(f'{COUNTRY_MICRODATA_DIR}/Datos.csv')
```

Comment:
No need to specify "Datos.csv" – if `load` is invoked with the directory it will load all csv files there.